students/ArchWu/Session8/mailroom_oo.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python3
+import sys, os, math
+from textwrap import dedent
+
+def init_donor_db():
+    db = DonorCollection()
+    db.add_donor(Donor("William Gates III", [653772.32, 12.17]))
+    db.add_donor(Donor("Jeff Bezos", [877.33]))
+    db.add_donor(Donor("Paul Allen", [663.23, 43.87, 1.32]))
+    db.add_donor(Donor("Mark Zuckerberg", [1663.23, 4300.87, 10432.0]))
+    return db
+
+class Donor:
```

Comment:
You already have this called in your donor models, you should import it instead.

Reply:
Actually this mailroom_oo is the file I was working on before I saw the requirement to split into multiple files and it happen to have the same file name as the requirement mentioned. What I did try is simply cutting those class initialization code into donor_model and split into two files.